### ISSUE TITLE
Failed to get EIP-1559 fees

### ISSUE THREAD

**Issue by c0deg33k** [Cyfrin/foundry-smart-contract-lottery-cu#53]
I'm trying to deploy my raffle contract to Sepolia TestNet and below is the terminal output I'm getting.
```
forge script script/DeployRaffle.s.sol:DeployRaffle --fork-url $SEPOLIA_RPC_URL --account sepolia1 --sender $SEPOLIA1_ADDRESS --broadcast
[⠊] Compiling...
No files changed, compilation skipped
Script ran successfully.

== Return ==
0: contract Raffle 0xDf3f8A545B5397726C00F14DCE0FBf76A051e056
1: contract NetworkChainConfig 0xC7f2Cf4845C6db0e1a1e91ED41Bcd0FcC1b0E141

== Logs ==
  Creating a new subscription on chainID:  11155111
  Here's your Subscription Id:  11924
  Funding Subscription:  11924
  Using vrfCoordinator:  0x8103B0A8A00be2DDC778e6e7eaa21791Cd364625
  On ChainId:  11155111
  Adding Consumer to contract:  0xDf3f8A545B5397726C00F14DCE0FBf76A051e056
  Using vrfCoordinator:  0x8103B0A8A00be2DDC778e6e7eaa21791Cd364625
  On chain Id:  11155111

## Setting up 1 EVM.
Error: 
Failed to get EIP-1559 fees
```
Anyone experienced the same?

**Comment by cromewar** [comment]
Hello @c0deg33k, sometimes the sepolia testnet does not work or it is slow, so before thinking it is a problem with one of the addresses you are using for VRF, can you try again? 

If the problem persists, let us know.

**Comment by PatrickAlphaC** [comment]
Usually this is an issue with the RPC URL. Could you try someone else? Maybe QuickNode? Are you using Alchemy? 

Going to close, but feel free to follow up.

**Comment by c0deg33k** [comment]
Yes I'm using alchemy. I looked into the issue and even after changing to QuickNode the error didn't go away. I however decided to clean my repo and it worked. I got the current version of every lib now and I actually just deployed [Raffle.sol](https://sepolia.etherscan.io/address/0x22db75248299fd2b0d36fb3980667738e7cedd8f). I can now sigh in relief.

> Usually this is an issue with the RPC URL. Could you try someone else? Maybe QuickNode? Are you using Alchemy?
> 
> Going to close, but feel free to follow up.
